egograph/pipelines/sources/browser_history/storage.py

```
@@ -100,25 +104,35 @@ def save_parquet(
         *,
         year: int,
         month: int,
-        prefix: str = datasets.BROWSER_HISTORY_PAGE_VIEWS.path,
+        dataset: DatasetDefinition = datasets.BROWSER_HISTORY_PAGE_VIEWS,
     ) -> str | None:
-        """events parquet を保存する。"""
+        """events parquet を保存する。
+
+        アップロード前に dataset の schema 契約（必須カラム・型）を検証し、
```

Ruff (0.16.0)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)